cms/templates/js/course-outline.underscore

```diff
@@ -116,6 +119,13 @@ if (xblockInfo.get('graded')) {
                         <% } %>
                     </span>
                 </p>
+            <% } else if (xblockInfo.getExamScore() !=null) { %>
+                <p>
+                    <span class="status-release-value">
```

Comment:
@marcotuts  I used only the '.status-exam' css . appended with .status-release rules.